### PR TITLE
fix(Tooltip): convert to TypeScript

### PR DESCRIPTION
**Pull request by tujoworker**
Next, we can convert it to use React Hooks along with testing lib 👍

### CODE REVIEW

File: packages/dnb-eufemia/src/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -4,101 +4,36 @@
...
-    hide_delay: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    target_selector: PropTypes.string,
-    target_element: PropTypes.oneOfType([
-      PropTypes.object,
-      PropTypes.node,
-    ]),
-
-    ...spacingPropTypes,
-
-    class: PropTypes.string,
```
Comment (langz) [inline packages/dnb-eufemia/src/components/tooltip/Tooltip.tsx:67]: Is it safe to remove this, not sure if found it in [TooltipProps](https://github.com/dnbexperience/eufemia/blob/452e889307c7420aafc179b0287ebc6de5a08987/packages/dnb-eufemia/src/components/tooltip/types.ts)

File: packages/dnb-eufemia/src/components/tooltip/TooltipContainer.tsx [hunk 2]
```diff
@@ -4,62 +4,33 @@
...
-    fixed_position: null,
-    no_animation: null,
-    useHover: true,
-    style: null,
-    attributes: null,
-    children: null,
-  }
+type TooltipContainerState = {
+  w?: number
+  h?: number
```
Comment (langz) [inline packages/dnb-eufemia/src/components/tooltip/TooltipContainer.tsx:23]: At a glance, I am not 100% sure what `w` and `h`, could we give these props any more meaningful names?

File: packages/dnb-eufemia/src/components/tooltip/Tooltip.tsx (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/tooltip/Tooltip.tsx:67]: `spacingPropTypes` is basically the same as `Props & ISpacingProps` – while `class` is not documented, and is a Web Component feature, which is, from what I know, not used in prod.

**Review by langz — APPROVED** [review]

File: packages/dnb-eufemia/src/components/tooltip/TooltipContainer.tsx (hunk 2 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/components/tooltip/TooltipContainer.tsx]: yes 🙏 I will rename them to bodyWidth, because that is what it is.

**Pull request merged**